Makefile

```diff
@@ -11,15 +11,31 @@ LDFLAGS=-ldflags "-w -s -X 'main.Version=${VERSION}' -X 'main.BuildTime=$(BUILD_
 
 DATABASE_ADDRESS?=bolt://localhost:7687
 
+PHONY: all
```

Comment:
Does `PHONY` need to be prefixed with a `.`? Most repos have the prefix, though I also noticed that the `dp-api-router` doesn't.

Reply:
I think it does need to be prefixed with a `.` - I will update here and in `dp-api-router`, too